README.md

```diff
@@ -15,42 +15,97 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-[![Apache Accumulo][logo]][accumulo]
---
-[![Build Status][ti]][tl] [![Maven Central][mi]][ml] [![Javadoc][ji]][jl] [![Apache License][li]][ll]
-
-[Apache Accumulo][accumulo] is a sorted, distributed key/value store that provides robust,
-scalable data storage and retrieval. With Apache Accumulo, users can store and manage large
+<!-- LOGO -->
```

Comment:
I don't see any reason to change the the markdown formatting to raw HTML, or change the layout/style of badges or the logo. And, there definitely shouldn't be a change to link to vuejs's site. I assume that was copied/pasted from another project and is a mistake, but I don't think any of this should be changed at all.

Reply:
> I don't see any reason to change the the markdown formatting to raw HTML, or change the layout/style of badges or the logo. 
 
The main idea is to make it look like this:
![image](https://user-images.githubusercontent.com/23703592/116788862-aa202280-aab4-11eb-8e12-c4f93739098e.png)

The only thing that has been modified is to make it look centered (and with that it would become an HTML format) the rest remains unchanged

---
> And, there definitely shouldn't be a change to link to vuejs's site. I assume that was copied/pasted from another project and is a mistake, but I don't think any of this should be changed at all.

At the moment VueJs does not sponsor Apache ... Although I never know ... 
_This kind of 'slip' is my signature, my apologies!_
It would be a matter of modifying the link .. :)

Reply:
I think it's better to stick with the Markdown instead of the HTML. I don't think centering is necessarily better or worth it. The Markdown is easier to maintain, and is the convention for many projects on GitHub, especially for the badges.

Reply:
In my humble opinion regarding maintenance (in this case) there would be no problem because the images are referenced in an external link and therefore it does not matter to use markdown or html.

>

Reply:
> I don't think centering is necessarily better or worth it.

Honestly, I see it as more minimalist, detailed ... _Hahaha I appeal to democracy ... would there be a way to vote?_

> The Markdown is easier to maintain, .(...) , especially for the badges.

In my humble opinion regarding maintenance (in this case) there would be no problem because the images and badges are referenced in an external link and therefore it does not matter to use markdown or html. Another thing is that there is behind a link automation or something similar.

> and is the convention for many projects on GitHub

There are projects like those of [Facebook](https://github.com/facebook/react-native), [Vue Js](https://github.com/vuejs/vue) that use html in their README

Reply:
@nicolasalarconrapela We don't just use Markdown because it's shorthand for HTML. We also use it because it's more human-readable. The less HTML, the better. We don't need HTML. This README is also included in the tarball we generate. So, it should be readable in a user's text editor, and the reader should not have to parse HTML/XML tags to read it. Centering the image, in my opinion, does not justify adding HTML that is harder for users to view using a plain text viewer.

Reply:
Perfect !!! I think it's great !!